README.md

```diff
@@ -43,6 +43,14 @@ It's also important to keep in mind that the `compile_frontend` management comma
 
 ### Loading FEC's regulations
 
+When there is new data available (e.g. due to modifications in the parser, new
+Federal Register notices, etc.), that data must be sent to the `/api` endpoint
+before it will be visible to users. However, we don't want to allow the
+general public to modify the regulatory data, so we need to authenticate.
+Currently, this is implemented via HTTP Basic Auth and a very long user name
+and password (effectively creating an API key). See the `HTTP_AUTH_USER` and
+`HTTP_AUTH_PASSWORD` environment variables in cloud.gov for more.
```

Comment:
However, there is a limit on how long each of these values can be, correct?  Or has this been accounted for via another change/update?

Reply:
I don't know what the HTTP spec limit is for Basic Auth. By using `~/.netrc` instead of specifying `username:password@` in the URL, we are circumventing the https://github.com/requests/requests/issues/4168 issue.

Reply:
That's what I was getting at, because that limit is definitely below the HTTP spec limit (which I was trying to look up the other day and not having much luck - it's somewhere in the [RFC](https://tools.ietf.org/html/rfc2616)!)

Reply:
Ha, I stand corrected!  Per section [3.2.1 in the HTTP RFC](https://tools.ietf.org/html/rfc2616#section-3.2.1):

>...
>
>The HTTP protocol does not place any a priori limit on the length of
>a URI. Servers MUST be able to handle the URI of any resource they
>serve, and SHOULD be able to handle URIs of unbounded length if they
>provide GET-based forms that could generate such URIs. A server
>SHOULD return 414 (Request-URI Too Long) status if a URI is longer
>than the server can handle (see section 10.4.15).
>
>>   Note: Servers ought to be cautious about depending on URI lengths
>>   above 255 bytes, because some older client or proxy
>>   implementations might not properly support these lengths.

Reply:
TIL, `user:password` in URL is deprecated.
https://bugs.chromium.org/p/chromium/issues/detail?id=82250#c7

Reply:
I also did not know IE hasn't supported this for some time according to those comments... TIL indeed!